topics/programability.md

```diff
@@ -0,0 +1,93 @@
+# Redis Programability
+Programability in Redis allows the user to send functionality (scritps) into Redis that will run on the sever where the data leaves.
+This can reduce network hops and improve overal perfromance.
+It can also enrich the API provided by Redis with a user define API,
+and allow implementing a schema over Redis data structures (updates multiple keys together, keep keys in sync).
```

Comment:
```suggestion
Programability in Redis allows the user to send functionality (scripts) into Redis that will run on the server where the data lives.
This can reduce network hops and improve overall performance.
It can also enrich the API provided by Redis with a user-defined API,
and allow implementing a schema over Redis data structures (update multiple keys together, keep keys in sync).
```